docs/pbc/all/usercentrics/integrate-usercentrics.md

```diff
@@ -73,7 +30,7 @@ To integrate Usercentics:
 
 That's it. You have integrated the Usercentrics app into your store. The app changes to *Connected* on the apps catalog page. Now, the cookie consent dialogs should be displayed on every page of the Storefront.
 
-### Global settings configuration
+## Global settings configuration
 
 There are three ways to integrate Usecentrics: by direct integration, by the Usercentrics Smart Data Protector, and integration with Google Tag Manager. In the Spryker Back Office, you can select either Smart Data Protector or Google Tag Manager. At the same time, the Smart Data Protector is the preferred and default setting.
```

Comment:
```suggestion
There are three ways to integrate Usecentrics: a direct integration,  using the Usercentrics Smart Data Protector, or using the Google Tag Manager. In the Spryker Back Office, you can select either Smart Data Protector or Google Tag Manager. At the same time, the Smart Data Protector is the preferred and default setting.
```

Reply:
"a direct integration,  using the Usercentrics Smart Data Protector" might be confusing, as it can be understood as one option. However, direct integration and Usercentrics Smart Data Protector are two different options, we should emphasize this.